jax/_src/device_array.py

```diff
@@ -125,6 +126,7 @@ def _check_if_deleted(self):
     if self.device_buffer is deleted_buffer:
       raise RuntimeError("DeviceArray has been deleted.")
 
+  @annotate_function
```

Comment:
I'd strongly prefer we use a qualified name. `jax.profiler.annotate_function` or `profiler.annotate_function` has an immediately clear and obvious meaning. `annotate_function` does not.

Same everywhere.

Reply:
No problem, done.